packages/ui/src/QueryProvider/index.tsx

```diff
@@ -9,13 +10,23 @@ type QueryProviderWrapperProps = {
 }
 
 export function QueryProvider({ children, persister, queryClient }: QueryProviderWrapperProps) {
-  // children = (<>{children}<ReactQueryDevtools buttonPosition="bottom-left" /></>)
+  // Cypress runs in dev mode which means the devtools button is visible, which may break tests.
+  const isCypress = typeof window !== 'undefined' && (window as any).Cypress
```

Comment:
there's no reason to keep this in the function, it can be defined globally

Reply:
old vue habits die hard